test-common/src/main/kotlin/SøknadsbehandlingTestData.kt

```diff
@@ -90,6 +96,10 @@ fun søknadsbehandlingVilkårsvurdertInnvilget(
     ),
     saksbehandler: NavIdentBruker.Saksbehandler = no.nav.su.se.bakover.test.saksbehandler,
     clock: Clock = fixedClock,
+    søknadsbehandlingsHistorikk: Søknadsbehandlingshistorikk = nySøknadsbehandlingshistorikkAlleVilkårMedBosituasjonOgFradrag(
```

Comment:
Det føles litt rart at man kan definere denne. Da får vi en større mismatch mellom produksjons- og testkoden. Gjelder for hele fila.